src/app/legacy/containers/PodcastExternalLinks/index.jsx

```diff
@@ -123,7 +123,7 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
       {hasMultipleLinks ? (
         <StyledList role="list">
           {links.map(({ linkText, linkUrl }) => (
-            <StyledListItem dir={dir} key={linkText}>
+            <StyledListItem dir={dir} key={`${linkText}`}>
```

Comment:
is there a reason why template literal has to be used here?

Reply:
yeah, it was needed to prevent tests in `OnDemandAudioPage` breaking